__tests__/app.test.js

```diff
@@ -67,22 +65,54 @@ describe("GET api/articles/:article_id", () => {
         expect(body.article).toEqual(output);
       });
   });
-  test("should return 404 if id is valid but does not exist",()=>{
-    return request(app).get("/api/articles/1000").expect(404).then(({text})=>{
-     
-    
-      
-      expect(text).toEqual("Article not found")
-
-    })
-  })
-  test("should return 400 if id is not valid" ,()=>{
-    return request(app).get("/api/articles/bananas").expect(400).then(({text})=>{
-    
-    
-      
-      expect(text).toEqual("Article not valid")
-
-    })
-  })
+  test("should return 404 if id is valid but does not exist", () => {
+    return request(app)
+      .get("/api/articles/1000")
+      .expect(404)
+      .then(({ text }) => {
+        expect(text).toEqual("Article not found");
+      });
+  });
+  test("should return 400 if id is not valid", () => {
+    return request(app)
+      .get("/api/articles/bananas")
+      .expect(400)
+      .then(({ text }) => {
+        expect(text).toEqual("Article not valid");
+      });
+  });
+});
+describe("Get /api/articles", () => {
+  test("should return all articles with their properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const article = body.article;
+        console.log(article,"article");
+        article.find(
```

Comment:
I'd recommend using a forEach for this and checking the length of the array before to avoid false positive 👍